wp-includes/class-wp-service-workers.php

```diff
@@ -57,10 +57,13 @@ class WP_Service_Workers implements WP_Service_Worker_Registry_Aware {
 	 */
 	public function __construct() {
 		$components = array(
-			'configuration'      => new WP_Service_Worker_Configuration_Component(),
-			'navigation_routing' => new WP_Service_Worker_Navigation_Routing_Component(),
-			'precaching_routes'  => new WP_Service_Worker_Precaching_Routes_Component(),
-			'caching_routes'     => new WP_Service_Worker_Caching_Routes_Component(),
+			'configuration'            => new WP_Service_Worker_Configuration_Component(),
+			'navigation_routing'       => new WP_Service_Worker_Navigation_Routing_Component(),
+			'theme_asset_caching'      => new WP_Service_Worker_Theme_Asset_Caching_Component(),
+			'plugin_asset_caching'     => new WP_Service_Worker_Plugin_Asset_Caching_Component(),
```

Comment:
Just realized that this is missing a caching strategy for assets from `wp-includes`, for example `jquery`, `wp-embed`, and `wp-emoji-release`.